pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/etterlatte/maler/fraser/barnepensjon/innvilgelse/BarnepensjonInnvilgelseEnkelFraser.kt

```diff
@@ -39,43 +39,43 @@ object BarnepensjonInnvilgelseEnkelFraser {
                 val formatertNyesteUtbetalingsperiodeDatoFom = nyesteUtbetalingsperiodeDatoFom.format()
 
                 textExpr(
-                    Bokmal to "Du er innvilget barnepensjon fra ".expr() + formatertVirkningsdato + " fordi " + avdoedNavn + " er registrert død " + formatertDoedsdato + ". ",
-                    Nynorsk to "".expr(),
+                    Bokmal to "Du er innvilget barnepensjon fra ".expr() + formatertVirkningsdato + " fordi " + avdoedNavn + " er registrert død " + formatertDoedsdato + ".",
+                    Nynorsk to "Du er innvilga barnepensjon frå og med ".expr() + formatertVirkningsdato + " fordi " + avdoedNavn + " er registrert død "+ formatertDoedsdato + ".",
                     English to "".expr()
                 )
 
                 showIf(harFlereUtbetalingsperioder) {
                     textExpr(
-                        Bokmal to "".expr() + "Du får "+ beloep.format() + " kroner hver måned før skatt fra " + formatertNyesteUtbetalingsperiodeDatoFom + ". Se utbetalingsbeløp for tidligere perioder i vedlegg om etterbetaling.",
-                        Nynorsk to "".expr(),
+                        Bokmal to "Du får ".expr() + beloep.format() + " kroner hver måned før skatt fra " + formatertNyesteUtbetalingsperiodeDatoFom + ". Se utbetalingsbeløp for tidligere perioder i vedlegg om etterbetaling.",
+                        Nynorsk to "Du får ".expr() + beloep.format() + " kroner per månad før skatt frå og med " + formatertNyesteUtbetalingsperiodeDatoFom + ". Sjå utbetalingsbeløp for tidlegare periodar i vedlegget om etterbetaling.",
                         English to "".expr()
```

Comment:
Ikke engelsk oversettelse for `InnvilgelseEnkelFraser`?

Reply:
Engelsk hadde ikke kommet når jeg lagde PR. Har lagt inn nå